### DISCUSSION TITLE
Q: authentication: 'Domain Join'/'Enable Stored MEtrics' not available in cockpit. Why?

### DISCUSSION THREAD

**Discussion by RogerWeihrauch** [telekom-security/tpotce#773] [Q&A]
Hi to all

I just installed latest TpotCE ontop of latest Debian Buster (10.8) w/: '../install.sh --type=user'. I gave Debian a std-user and access to root via sudo before, which enabled me to successfully setup a TopotCE, incl. 2fa, which wall works. So, my question is:
How can I authorize this std-user to:
- join a domain
- enable stored metrics
- get access to Docker container overview from within cockpit?
This user also is not allowed to shutdown/restart from within cockpit and also fails to obtain authentication to execute a software update from within cockpit.
Why is this that way? What did I miss to get those authentications?
What do I have to do to get those authentications to execute those commands?
Thank you all for any ongoing hints on this.
Kind regards,
Roger

**Comment by t3chn0m4g3** [comment]
From a security perspective I am recommending against a domain join, however you may want to look at the cockpit [documentation ](https://cockpit-project.org/guide/172/feature-realmd.html) for further instructions.
